### PR TITLE
Week1 homework

### PR DESCRIPTION
**Pull request by abdullahkanatsiz7**
This is my homework-1. Sorry I'm too late.I was so busy. But I will complete the other homework as soon as possible short time.

### CODE REVIEW

File: Week1/databases/Homework/package.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (roxana-lafuente) [inline Week1/databases/Homework/package.json]: Great for adding the package.json!

File: Week1/databases/Homework/index.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  sqlCreateWorld,
+  sqlCreateCountry,
+  sqlCreateCity,
+  sqlWorld,
+  selectQueries,
+  insertCityData,
+  insertCountryData,
+} = require('./query');
+
+const Query = mysql.createConnection('mysql://hyfuser:hyfpassword@192.168.153.132/userdb');
```
Comment (roxana-lafuente) [inline Week1/databases/Homework/index.js]: I guess that's the IP of your own personal computer? Usually what we do is to point to `localhost` or `127.0.0.1` which means it points to the machine where the code is being run.

File: Week1/databases/Homework/index.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+const startQuery = util.promisify(Query.query.bind(Query));
+
+(async () => {
+  Query.connect();
+  try {
+    await startQuery(sqlCreateWorld);
```
Comment (roxana-lafuente) [inline Week1/databases/Homework/index.js]: Just a note here:

You create `world` database but you're assuming that `userdb` already exist.

```
const Query = mysql.createConnection('mysql://hyfuser:hyfpassword@192.168.153.132/userdb');
```

File: Week1/databases/Homework/index.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
+'use strict';
```
Comment (roxana-lafuente) [inline Week1/databases/Homework/index.js]: Usually it's a good idea to add a .md file with instructions on how to use your project, this file usually has:

- dependencies, maybe how to install them
- how to use your code ( how to run it, if you have provide an API, how to use, which parameters are required, etc)
- etc

File: Week1/databases/Homework/query.js [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+const insertCityData = `INSERT INTO city (Name, CountryCode, District, Population) VALUES ('Amsterdam','NLD','Noord-Holland',821752), ('Ottawa','CAN','Ontario',934243), ('Berlin','DEU','Berliini',4105000),  ('Kingston','JAM','St. Andrew',666041), ('Hanoi','VNM','Hanoi',7782000);`;
+
+const selectQueries = [
```
Comment (roxana-lafuente) [inline Week1/databases/Homework/query.js]: If I run this, nothing happens. I don't see where these queries are being used.

**Comment by Tjebbee** [comment]
@abdullahkanatsiz7 Can you please reply on this?

**Pull request closed without merging**